internal/generate/handle_generate.go

```diff
@@ -133,13 +133,17 @@ func (s *Server) generateKeysInRegion(ctx context.Context, region string) error
 		if err != nil {
 			return fmt.Errorf("failed to decide revised key status: %w", err)
 		}
-		generateRevisedKeys := val <= s.config.ChanceOfKeyRevision
+		generateRevisedKeys := val < s.config.ChanceOfKeyRevision
 
 		reportType := verifyapi.ReportTypeClinical
 		if !generateRevisedKeys {
-			reportType, err = util.RandomReportType()
-			if err != nil {
-				return fmt.Errorf("failed to generate report type: %w", err)
+			if forceConfirmed := s.config.ForceConfirmed; forceConfirmed {
```

Comment:
this should just be
`if s.config.ForceConfirmed`

no reason to assign to another var

Reply:
ah yep thanks, was leftover from some other testing